proarc-webapp/src/main/java/cz/cas/lib/proarc/webapp/client/widget/workflow/WorkflowJobFormView.java

```diff
@@ -413,7 +413,7 @@ public void performAction(ActionEvent event) {
     }
 
     private Widget createMaterialList() {
-        materialView = new WorkflowMaterialView(i18n, true);
+        materialView = new WorkflowMaterialView(i18n, true, this.getClass().getSimpleName());
```

Comment:
Martine, co vrací tahle reflection po překladu do javascriptu? Ten prefix by bylo dobré mýt čitelný. Není to nějaký optimalizovaný identifikátor?